docs/install-include.rst

```diff
@@ -107,18 +127,42 @@ Select your preferences and run the install command.
 
            .. container:: cpu
 
+              .. note::
+              
+                 If you don't have them, you must first install the following packages: 
+                 [XCode](https://developer.apple.com/xcode/), [Homebrew](https://brew.sh/), [opencv](https://opencv.org/), [LibOMP](https://formulae.brew.sh/formula/libomp)
+                 Once you have Homebrew, the latter two packages can be installed via:
+
+                 .. code-block:: bash
+
+                     brew install libomp opencv
+
               .. code-block:: bash
 
                  pip install --upgrade mxnet-osx
                  git clone https://github.com/awslabs/autogluon
                  cd autogluon && python setup.py install --user
 
-           .. container:: cuda
+           .. container:: gpu
+
+              .. note::
+              
+                 If you don't have them, you must first install the following packages: 
+                 [XCode](https://developer.apple.com/xcode/), [Homebrew](https://brew.sh/), [opencv](https://opencv.org/), [LibOMP](https://formulae.brew.sh/formula/libomp)
+                 Once you have Homebrew, the latter two packages can be installed via:
+
+                 .. code-block:: bash
 
-              Please build MXNet from source manually, details in `MXNet Docs<https://mxnet.apache.org/get_started?version=v1.5.1&platform=macos&language=python&environ=build-from-source&processor=gpu>`_.
+                     brew install libomp opencv
+
+              Please build MXNet from source to utilize GPU, following detailed instructions from the [MXNet Docs](https://mxnet.apache.org/get_started?version=v1.5.1&platform=macos&language=python&environ=build-from-source&processor=gpu).
 
               .. code-block:: bash
 
                  git clone https://github.com/awslabs/autogluon
                  cd autogluon && python setup.py install --user
 
+  .. note::
+
+     AutoGluon is only supported for [Python](https://www.python.org/downloads/release/python-370/) versions >= 3.6. Make sure [CUDA](https://developer.nvidia.com/cuda-downloads) and [cuDNN](https://developer.nvidia.com/cudnn) are installed if you want to use GPU.
```

Comment:
This makes the website a little bit verbose http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-133/4/index.html

Users won't be able to install autogluon if python version < 3.6 https://github.com/awslabs/autogluon/blob/0a0fa6307314ee5b6dc13eb74bddee58daed5691/setup.py#L86
Do we still need this?

Reply:
@Innixma what's your thoughts?

Reply:
Looks like the links aren't properly applied. Can you fix this along with the GPU typo Hang? I think it will look fine once the links are properly applied.

Reply:
its much shorter now with the links properly formatted